styles/style.css

```diff
@@ -96,3 +96,153 @@ body {
     margin-top: 40px;
 }
 
+/* Styles for main section*/
+.main-container {
+    min-width: 1440px;
+    min-height: 100vh;
+    padding: 51px 104px 120px;
+    margin: 0 auto;
+    background: url(../img/project_images/mainpage-image.png) no-repeat center center;
+}
+
+.main-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main-header-logo {
+    width: 205px;
+    height: 40px;
+    display: block;
+    object-fit: contain;
+}
+
+.main-header-list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    position: relative;
+}
+
+.main-header-items {
+    color: #FFFFFF;
+    font-size: 24px;
+    font-weight: 400;
+    margin-left: 24px;
+}
+
+.main-header-items:first-child::before {
+    content: "";
+    position: absolute;
+    width: 50px;
+    height: 4px;
+    top: 100%;
+    left: 8%;
+    background-color: #F5BD41;
+    margin-top: 4px;
+}
+
+.night-icon {
+    background: url(../img/project_images/night-icon.png) no-repeat center center;
+    width: 30px;
+    height: 30px;
+    margin-left: 48px;
+}
+
+.account-icon {
+    background: url(../img/project_images/account-icon.png) no-repeat center center;
+    width: 40px;
+    height: 40px;
+    margin-left: 24px;
+}
+
+.main-title {
+    max-width: 542px;
+    font-size: 50px;
+    font-weight: 500;
+    line-height: 59px;
+    color: #FFFFFF;
+    margin-top: 202px;
+    margin-bottom: 100px;
+}
+
+.main-search {
+    margin-bottom: 120px;
+}
+
+.search-form {
+    display: flex;
+    align-items: flex-end;
+}
+
+.search-input {
+    border: 3px solid #F3F3F4;
+}
+
+.label {
+    padding-left: 24px;
+}
+
+.label-search {
+    color: #FFFFFF;
+    font-size: 18px;
+    font-weight: 400;
+    display: inline-block;
+}
+
+.search-input[placeholder] {
+    padding: 21px 24px 22px;
+    font-size: 18px;
+    font-weight: 400;
+    margin-top: 24px;
+}
+
+.input-place {
+    min-width: 458px;
+    border-radius: 8px 0 0 8px;
+}
+
+.input-date {
+    min-width: 256px;
+    padding-right: 24px;
+    border: 3px solid #F5BD41;
```

Comment:
You should add a focus with orange border for inputs, and remove outline for focus.
And it's better to add a white background for a wrapper, because I see empty spot, it's not according to design
<img width="1376" alt="Screenshot 2022-03-11 at 23 33 14" src="https://user-images.githubusercontent.com/25488411/157958799-76473f6a-332c-4c7b-8aca-f5e7732544be.png">

Reply:
Done. I added :: before to wrapper. I am not sure how it will work for responsive design.